### PR TITLE
corrected the annotations

### PR DESCRIPTION
**Pull request by akun1012**

### CODE REVIEW

File: pkg/cgroups/cgroups.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ import (
 )
 
 const (
-	CPUPeriod = 1000000 // one hundred thousand microseconds
+	CPUPeriod = 1000000 // one second
```
Comment (esuen) [inline pkg/cgroups/cgroups.go]: Is there reason why we don't want to specify the unit is in microseconds?

Comment (akun1012) [reply pkg/cgroups/cgroups.go]: nope, the change is not about changing unit from microsecond to second, it's about correcting it from "one hundred thousand microseconds" to "one million microseconds" which is one second.

Comment (esuen) [reply pkg/cgroups/cgroups.go]: Maybe put "one million microseconds which equals 1 second".

Comment (ausmith) [reply pkg/cgroups/cgroups.go]: +1 to @esuen's suggestion of making it explicit.

Comment (akun1012) [reply pkg/cgroups/cgroups.go]: polished

**Pull request merged**